### ISSUE TITLE
Question: How to interpret the hex2vec embedder training loss?

### ISSUE THREAD

**Issue by adesso-dominik-chodounsky** [kraina-ai/srai#449]
I am using the hex2vec embedder and upon looking at the training loss, I see that no matter what hyperparameters I use, the first epoch step has a loss larger than 1.0 (e.g. 1.5) and all the next steps have exactly 1.0.

For f-score, the metric gradually increases, so I guess the model is getting better, but the loss stays exactly the same. Can anyone explain this behaviour please?
